google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java

```diff
@@ -87,6 +87,14 @@
         required = true,
         description = "The ID of a store sales conversion action")
     private Long conversionActionId;
+    
+    @Parameter(
+        names = ArgumentNames.CUSTOM_KEY,
+        required = false,
+        description = "Only required after creating a custom key and custom values in the account."
+                + " Custom key and values are used to segment store sales' conversions."
```

Comment:
nit: remove apostrophe on "sales". I don't think you need that to be possessive.

Reply:
Done.